stack.yaml

```diff
@@ -40,6 +40,6 @@ extra-deps:
   - git: https://github.com/kadena-io/merkle-log.git
     commit: a7ae61d7082afe3aa1a0fd0546fc1351a2f7c376
   - git: https://github.com/kadena-io/pact.git
-    commit: 9456a750f67b62d48bed5ef2e60fbc7e9d2a4396
+    commit: b5601c601c3e497c938f6d804eb21f9a3fb30a2a
```

Comment:
This is the commit from the PR. The merged commit is: 0915aff9eed762b2ee58c648d900bc619e6fff96